src/byteStream.js

```diff
@@ -76,6 +76,10 @@ export default class ByteStream {
     return new ByteStream(this.byteArrayParser, byteArrayView);
   }
 
+  getSize() {
```

Comment:
I was wondering if this is necessary change. I'm telling that because with that change there will one place using getSize and several other places using the full one (byteArray.length)

Reply:
I'd rather have a function - ideally it would be a getter, which I might change it to, but having it know about hte internals isn't a good idea.